README.md

```diff
@@ -189,14 +189,33 @@ For example:
 
 going to execute `eslint` and if it exits with `0` code, it will execute `prettier --write` on all staged `*.js` files.
 
-## Using JS functions to customize tasks
+## Using JS configuration file
 
-When supplying configuration in JS format it is possible to define the task as a function, which will receive an array of staged filenames/paths and should return the complete command as a string. It is also possible to return an array of complete command strings, for example when the task supports only a single file input. The function can be either sync or async.
+Writing `lint-staged.config.js` file (or, other JS file which needs to be mentioned in CLI command through `c` flag) is the most powerful way to configure `lint-staged`. From the configuration file, you can export a function which gets an array of all staged file paths as parameter and returns command(s) as string or array of string. This function can be either sync or async. Signature of this function:
```

Comment:
```suggestion
Writing the configuration file in JavaScript is the most powerful way to configure _lint-staged_ (`lint-staged.config.js`, similar, or passed via `--config`). From the configuration file, you can export either a single function, or an object.

If the `exports` value is a function, it will receive an array of all staged filenames. You can then build your own matchers for the files, and return a command string, or an array or command strings. These strings are considered complete and should include the filename arguments, if wanted.

It the `exports` value is an object, its keys should be glob matches (like in the normal non-js config format). The values can either be like in the normal config, or then individual functions like described above. Instead of receiving all matched files, the functions in the exported object will only receive the staged files matching the corresponding glob key.
```

Reply:
the `similar` word could actually link to the part about using cosmiconfig:

```md
[suggestion](https://github.com/okonet/lint-staged/README.md#configuration)
```

Reply:
This is much better, but there's a typo in the last line. `It` should be `If`.

Reply:
Good catch, fixed!

Reply:
I also put in the similar-link into the suggestion.